web/index.html

```diff
@@ -28,6 +28,11 @@
         input[type=text] {
             -webkit-user-select: text !important;
         }
+        input[type=date]::-webkit-calendar-picker-indicator {
+            position: absolute;
+            right: 12px;
+            top: 16px;
+        }
```

Comment:
This positions the datepicker calendar icon similar to how we do it on other fields (select picker)
- Edge: 
![Edge](https://user-images.githubusercontent.com/12156624/135151129-dc8113ae-d1e1-480c-8b5a-29fc3851757f.png)
- Chrome: 
![image](https://user-images.githubusercontent.com/12156624/135151185-4df04208-ad22-4837-931f-a18147211171.png)
- Firefox and Safari are not displaying an icon: 
![image](https://user-images.githubusercontent.com/12156624/135151580-92518c55-9ed8-4222-ade5-c8e5ed2b289b.png)